### PR TITLE
Multiple documentation updates

### LINKED ISSUE #683: Add fmrwhy to multi-echo resources
### Summary

Recently, @jsheunis introduced fmrwhy (https://fmrwhy.readthedocs.io/), a MATLAB package for processing multi-echo data. It would be great to include this in our documentation as a resource for the multi-echo community !

### Additional Detail

I think it would need to be added [here](https://tedana.readthedocs.io/en/stable/multi-echo.html#resources) under **Multi-echo processing software**, but pending #677 the exact text area could be updated.

### Next Steps

Open a PR to add this in :smile_cat:

### LINKED ISSUE #898: Improve documentation on masking options
### Summary

There have been several situations were the automasking that is built into tedana spectacularly fails. We should trace this down, but our sense is that many uses are providing their own masks anyway and that avoids this problem. We should add to the documentation make clear that this is a known issue that is resolved with user-provided masks.

### Additional Detail

- Based on conversation in [October 2022 Dev call](https://docs.google.com/document/d/1dyuUarQNwsZ6DeZWIOMzUtWw3zMlXiUNJlQa_lp5xbw/edit?usp=sharing)
- Conversation focused on https://neurostars.org/t/visualisation-of-component-beta-maps-using-tedana/23658/3 (possible link to relevant section of code (https://github.com/ME-ICA/tedana/blob/c7924b502dd89dbd8f9c5fbc4581763c36b9dab8/tedana/utils.py#L78) but there have been other complaints about the built-in automask

### Next Steps

- Supplement documentation
- If there are potential code changes to improve how masking is handed in tedana, open a new issue for that

### LINKED ISSUE #900: Mention dropping non-steady-state volumes in documentation
Our documentation doesn't cover dropping dummy scans before running tedana. The gists we include for running fMRIPrepped data through tedana, for example, don't mention that users should probably identify the non-steady-state volumes from the fMRIPrep confounds file and drop those volumes from the echo-wise data before denoising.

> The highest variance components have a very large spike at the beginning of the scans. That is usually a sign that the non-steady state volumes weren't removed. Siemens removes those by default, but GE keeps them. Looking at the openneuro description of these data, some participants were acquired on a Siemens scanner and others on a GE scanner. Any chance this run was GE scanner without steady state volumes removed?

_Originally posted by @handwerkerd in https://github.com/ME-ICA/tedana/discussions/899#discussioncomment-4171093_

### LINKED ISSUE #942: Re-do interactive reports demo in documentation
### Summary

When #756 is merged, the interactive reports will change slightly. The plots will includes lines to show the kappa and rho elbows and the hover text will show classification tags. The explanation for the output text in `outputs.rst` should also change. This will require uploading a new interactive demo (currently linking to https://me-ica.github.io/tedana-ohbm-2020/ ) and redoing every screen shot.

### Next Steps

- Decide if there are other changes to make to the interactive reports before updating the documentation.
  - #764 mentions adding the elbow thresholds (now done) and also variance info.
  - #888 is more complex and not worth waiting for unless someone decides that's a fun side-project
- Upload a new interactive demo for the OHBM 2023 tedana poster and link to the demo in the docs
- Take screen shots of that demo and update the documentation text accordingly.

### PR DESCRIPTION
**Pull request by handwerkerd**
Closes #683. Closes #898. Closes #900. Closes #942 

This will be a PR to address many of the open issues regarding documentation. As I address more, I'll update this initial message. Others are welcome to work on documentation issues/gap and add to this PR.

Changes proposed in this pull request:

- Adds a reference to ``fmrwhy`` in ``multi-echo.rst`` (#683)
- Cleaned up text on distortion correction and spatial normalization in ``multi-echo.rst`` (follow-up to #896)
- Edited text on the adaptive mask in ``approach.rst`` and added "Why are there so few voxels in my mask?" to ``faq.rst`` (#898)
- Edited text and replaced static figures in ``outputs.rst`` ica-components-report. Also linking to the new interactive demo from OHBM2023. (#942)
  - In making the above edits, I also added text on how to interpret each of the report figures. This might also address #915 but I'm not (yet) going to close it with this PR, pending feedback from others about whether we would include more information elsewhere to fully address #915.
- Added info about non-steady state volumes to a note in the initial "Multi-echo data" section of ``approach.rst`` and as a issue to look for when looking at time series in the reports in ``outputs.rst``  "individual-component-view" (#900)

### CODE REVIEW

**Comment by tsalo** [comment]
@all-contributors please add @handwerkerd for code.

**Comment by handwerkerd** [comment]
I think I've included all the open low-hanging-fruit documentation open issues. There are a few others that might take a bit more work and are sufficiently separate from the rest of these. It would be good to get reviews from both people who know what's happening under the hood and people who can see if I'm explaining things clearly. While you're reading, also keep an eye out for other simple things that could be written more clearly or have added info. @tsalo @eurunuela @dowdlelt @n-reddy @NaomiGaggi @marco7877

**Review by tsalo — APPROVED** [review]
There might be a small typo, but LGTM otherwise.

File: docs/approach.rst
Comment (dowdlelt) [inline docs/approach.rst:59]: Is it always divided by 3, or is it N_Echoes? I can also just check adaptive masking...

**Review by tsalo — DISMISSED** [review]

**Review by dowdlelt — APPROVED** [review]
github is being finicky...I approve pending typo corrections

File: docs/faq.rst
Comment (handwerkerd) [reply docs/faq.rst:80]: All voxels with at least 1 good echo are included. The revised text clarifies this.

**Review by dowdlelt — APPROVED** [review]
I abused my powers to fix a typo, because honestly it feels like kind of insane to point at a typo, say I approve, get that dismissed, then approve again. I approve!

**Review by tsalo — APPROVED** [review]

**Pull request merged**
